apps/students/views.py

```diff
@@ -173,6 +173,15 @@ def ajax_attendance(request):
     return JsonResponse(data)
 
 
+def ajax_student_attendance(request):
+    stu_id = request.GET['std_id']
+    stu_att = StudentAttendance.objects.get(student__id=stu_id, cal_date=today_date)
+    stu_att.present = not stu_att.present
```

Comment:
Don't just toggle the attendance, be sure whether you want to mark the student present or absent. Bring in one more key-value pair for this in GET request. (Send True if the checkbox is checked and False if it is unchecked).